src/main/java/com/softserve/itacademy/exception/handler/GenericExceptionHandler.java

```diff
@@ -97,4 +94,16 @@
 
         return new ResponseEntity<>(dto, HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler({OperationNotAllowedException.class})
+    public ResponseEntity<BasicExceptionResponse> handleAuthException(OperationNotAllowedException exception) {
```

Comment:
You forgot to rename method)

Reply:
Thanks, will be fixed in `Cleaning up` commit